osxcollector/osxcollector.py

```diff
@@ -34,8 +34,8 @@
 from hashlib import sha256
 from json import dumps
 from numbers import Number
-from sqlite3 import connect
 from sqlite3 import OperationalError
+from sqlite3 import connect
```

Comment:
super minor - if we want imports to be alphabetized, connect should go before OperationalError.

If not, feel free to reject this comment.